### ISSUE TITLE
[BUG/ISSUE] Obspack diagnostic never initializes in master branch

### ISSUE THREAD

**Issue by ltmurray** [geoschem/geos-chem#340]
The following needs to be added circa line 295 of input_mod.F90, otherwise ObsPack is never initialized.

```
       ELSE IF ( INDEX( LINE, 'OBSPACK MENU'     ) > 0 ) THEN
          CALL READ_OBSPACK_MENU( Input_Opt, RC )
          IF ( RC /= GC_SUCCESS ) THEN
             ErrMsg = 'Error in "Read_Obspack_Menu"!'
             CALL GC_Error( ErrMsg, RC, ThisLoc )
             RETURN
          ENDIF

```

**Comment by msulprizio** [comment]
Thanks, Lee! We just fixed this issue and it will be included in 12.9.0. See https://github.com/geoschem/geos-chem/issues/337.

**Comment by ltmurray** [comment]
Sorry for missing issue 337!
